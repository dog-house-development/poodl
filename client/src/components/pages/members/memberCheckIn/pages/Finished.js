import React, { Component } from 'react';

class Finished extends Component {
    render() {
        return (
            <div className="panel">
                <h2 className="panel-title">All done!</h2>
                <p>Have a great day!</p>
            </div>
        );
    }
}

export default Finished;
