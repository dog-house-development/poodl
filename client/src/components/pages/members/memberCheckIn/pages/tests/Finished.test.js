import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import _ from 'lodash';

import Finished from '../Finished';

configure({ adapter: new Adapter() });

describe('Button tests', () => {
    let wrapper, instance;
    const setInstanceAndWrapper = () => {
        wrapper = shallow(<Finished />);
        instance = wrapper.instance();
    };

    beforeEach(() => {
        setInstanceAndWrapper();
    });

    describe('render', () => {
        it('should render correctly', () => {
            expect(wrapper).toMatchSnapshot();
        });
    });
});
