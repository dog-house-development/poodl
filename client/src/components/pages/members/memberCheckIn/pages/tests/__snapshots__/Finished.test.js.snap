// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests render should render correctly 1`] = `
<div
  className="panel"
>
  <h2
    className="panel-title"
  >
    All done!
  </h2>
  <p>
    Have a great day!
  </p>
</div>
`;
